en/flight_controller/crazyflie2.md

```diff
@@ -94,19 +94,19 @@ For systems that support [make](https://www.gnu.org/software/make/manual/html_no
  * `make venv`
  * Activate the virtual environment: `source venv-cflib/bin/activate`
 
-To launch cfbridge.py everytime:
+To **launch cfbridge** everytime:
 - Switch on CF2 (which is already flashed with PX4 firmware) by pressing its ON button and wait for it to boot up.
 - Connect a Crazyradio PA via USB.
 - Navigate to the crazyflie-lib-python folder.
 - Activate the environment: `source venv-cflib/bin/activate` or `source venv/bin/activate` depending on the name of the venv folder you have.
 - `cd examples`
-- `python cfbridge.py`
+- Launch cfbridge: `python cfbridge.py`
 - Open QGC.
 - After using cfbridge, you can deactivate the virtualenv if you activated it by pressing `CTRL+z`. Most of the time, launching cfbridge again from the same terminal doesn't connect to crazyflie, this can be solved by closing the terminal and relaunching cfbridge in a new terminal. 
 
-> **Note** If you change any driver in [crazyflie-lib-python](https://github.com/barzanisar/crazyflie-lib-python/tree/cfbridge) 
+> **Tip** If you change any driver in [crazyflie-lib-python](https://github.com/barzanisar/crazyflie-lib-python/tree/cfbridge) 
```

Comment:
which significant changes do you have in your fork? should we clean then and make a pull request to the crazyflie repo?

Reply:
@barzanisar IMO this shouldn't be merged if it is reliant on any private forks.

Reply:
I have now linked to the original cflib-python.